### PR TITLE
Add Phase 5

### PR DESCRIPTION
**Pull request by kgrsutos**
## Summary by CodeRabbit

* **New Features**
  * CloudWatch log retrieval now filters logs to include only those containing "Started" or "Completed", improving efficiency and reducing data transfer.

* **Documentation**
  * README and CLAUDE.md were extensively rewritten and expanded with detailed English documentation, installation, usage, architecture, and development guidelines.
  * Requirements and implementation docs updated for clarity, expanded path parameter normalization, and revised risk mitigation strategies.

* **Bug Fixes**
  * Clarified output format and log filtering requirements in documentation.

* **Tests**
  * Added and enhanced tests for CloudWatch log filtering, pagination, error handling, and command-line flag validation.
  * Improved test coverage for time parsing, command integration, and edge cases.

* **Refactor**
  * Removed the ability to instantiate the aggregator with a custom configuration file.

### CODE REVIEW

File: internal/cloudwatch/client.go [hunk 1]
```diff
@@ -50,10 +50,16 @@ func NewClientWithAPI(api CloudWatchLogsAPI) *Client {
...
+	
 	input := &cloudwatchlogs.FilterLogEventsInput{
-		LogGroupName: &logGroupName,
-		StartTime:    int64Ptr(startTime.UnixMilli()),
-		EndTime:      int64Ptr(endTime.UnixMilli()),
+		LogGroupName:  &logGroupName,
+		StartTime:     int64Ptr(startTime.UnixMilli()),
+		EndTime:       int64Ptr(endTime.UnixMilli()),
+		FilterPattern: &filterPattern,
 	}
```
[the remark this answers is not in the document]
Comment (kgrsutos) [reply internal/cloudwatch/client.go:63]: ?Started ?Completed この構文で正しいです。` ||` はサポートされていません。
